src/content/guidelines/accessibility/color.md

```diff
@@ -40,18 +39,18 @@ Low vision can include partial sight in one or both eyes, and range from mild to
 | Type | Color deficiency |
 |------| ----------------|
 | Low acuity | Also known as **blurred vision.** Can make text difficult to read, since it would appear fuzzy. |
-| Low contrast sensitivity | The ability to determine fine detail and distinguish one object from another.     |
+| Low-contrast sensitivity | The ability to determine fine detail and distinguish one object from another.     |
 | Obstructed visual field | The user's view is partially obstrucuted. Can include central vision and spotty vision.  |
 | Retinitis Pigmentosa | Also known as **tunnel vision**. Obstructed peripheral vision - user is only able to see central elements  |
 
 _[Source: NoCoffee - Visual impairments ](https://accessgarage.wordpress.com/2013/02/09/458/)_
 
 ![blurred vision and tunnel vision on Google.com](images/color-accessibility-2.png)
 
-_Low vision disabilities_
+_Low-vision disabilities_
 
 ### Designing for low vision
-To get an understanding of the various low vision disabilities, we recommend using the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites. Low vision users may be using a screen reader to preview your website or experience, so ensure you're following [keyboard guidelines](/guidelines/accessibility/keyboard) to ensure the page is read to the user in a logical order.
+To get an understanding of the various low-vision disabilities, we recommend using the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites. Low-vision users may be using a screen reader to preview your website or experience, so ensure you're following [keyboard guidelines](/guidelines/accessibility/keyboard) to ensure the page is read to the user in a logical order.
```

Comment:
Would be fun to plug http://www.getstark.co/ here, as well.

Reply:
It's in there.